openc3/lib/openc3/models/scope_model.rb

```diff
@@ -379,9 +379,9 @@ def seed_database
       setting = SettingModel.get(name: 'source_url')
       SettingModel.set({ name: 'source_url', data: 'https://github.com/OpenC3/cosmos' }, scope: @scope) unless setting
       setting = SettingModel.get(name: 'rubygems_url')
-      SettingModel.set({ name: 'rubygems_url', data: 'https://rubygems.org' }, scope: @scope) unless setting
+      SettingModel.set({ name: 'rubygems_url', data: ENV['RUBYGEMS_URL'] }, scope: @scope) unless setting
       setting = SettingModel.get(name: 'pypi_url')
-      SettingModel.set({ name: 'pypi_url', data: 'https://pypi.org' }, scope: @scope) unless setting
+      SettingModel.set({ name: 'pypi_url', data: ENV['PYPI_URL'] }, scope: @scope) unless setting
```

Comment:
These should have a default if the ENV isn't set.

Reply:
I adjusted this so now the default is `https://pypi.org` if `ENV['PYPI_URL']` is not set.